CHANGES.md

```diff
@@ -1,10 +1,142 @@
 # GAP - history of changes
 
-## GAP 4.11.0 (February 2020)
+<!--
+
+The following is an unstructured list of all those merged pull requests
+for GAP 4.12.0 until February 17th, 2021, that are relevant for release notes.
+The label "release notes: added" has already been attached to them.
+
+### New features and major changes
```

Comment:
Reminder: when we backport this to stable-4.11, don't forget to remove this section